### DISCUSSION TITLE
Deep Link for searches and tags

### DISCUSSION THREAD

**Discussion by extrafein** [usememos/memos#3999] [Q&A]
Hi,

is there something like deep links in memos?
This would make any of my intergartions with other tools a little bit tighter.

Im searching something like:

> https://memos.myhost.com/?tags=urgent

Thank you.

**Comment by RoccoSmit** [comment]
The feature will be in the next release. Commit reference: https://github.com/usememos/memos/pull/4239

**Comment by RoccoSmit** [comment]
its in v0.23.1
